blockmanager.go

```diff
@@ -1040,7 +1061,12 @@ func (b *blockManager) getCheckpointedCFHeaders(checkpoints []*chainhash.Hash,
 			// Then, we cycle through any cached messages, adding
 			// them to the batch and deleting them from the cache.
 			for {
-				checkPointIndex++
+				// Determine the next checkpoint index we should
+				// process.
+				checkPointIndex += 2
```

Comment:
Can lift the `2` value used throughout the diff into something like a `checkpointsPerMaxQuery` constant perhaps.